src/lib/coreboot_table.c

```diff
@@ -572,6 +572,10 @@ static uintptr_t write_coreboot_table(uintptr_t rom_table_end)
 	/* Add board-specific table entries, if any. */
 	lb_board(head);
 
+	/* Add information about firmware in form suitable for EFI updates. */
```

Comment:
Maybe do this before `lb_board()`? That function is most likely the place where other components will be added, it seems logical to have main FW as first entry.

Reply:
Moved: https://github.com/Dasharo/coreboot/compare/00bebe0e391889f56d2891b310d7a7250680c021..702c924defd5047017c5ea5f154e41f0d96b7d02#diff-ceee6c791f39074f69b3cc196251618d0d22a8c3c7227f6b44783c08ec69b51f